### ISSUE TITLE
Fileupload in repeating group contrast is too low

### ISSUE THREAD

**Issue by Magnusrm** [Altinn/app-frontend-react#4004]
### Description of the bug

Since the fileupload is an interactive element the contrast should be at least 3:1 according to accessability guidelines. Here the contrast between the fileupload border and the background is 2.38:1 which is too low.
We should make use of the designsystems design tokens in order to ensure sufficient contrast and enable use of themes and dark mode in the future.

<img width="932" height="267" alt="Image" src="https://github.com/user-attachments/assets/02a88fe3-58d7-4f2f-95cf-7b4eac4f3bfd" />

### Steps To Reproduce

1. go to an app with a file upload in a repeating group ex. TTDs "frontend-test"
2. open the repeating group and see that the contrast is too low using color contrast tools like "Colour contrast checker"
